composer.json

```diff
@@ -18,19 +18,17 @@
     }
   ],
   "require": {
-    "php": "^7.1",
```

Comment:
In  arrayee you did so: `"php": ">=7.3",`

Reply:
gotcha

Reply:
fixed